corehq/apps/case_search/dsl_utils.py

```diff
@@ -12,6 +12,8 @@
 def unwrap_value(domain, node):
     """Returns the value of the node if it is wrapped in a function, otherwise just returns the node
     """
+    if isinstance(node, (str, int, float, bool)):
+        return node
```

Comment:
Curious what this achieves that the `if not isinstance(node, FunctionCall)` line doesn't catch?

Reply:
Good point. What I was wanting to do was only permit certain types here. For example it should not return a `BinaryExpression`. But on more reflection perhaps that doesn't belong here.

Reply:
72f84b49ea9